ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/CompactorUtil.java

```diff
@@ -74,6 +77,20 @@ static Runnable unchecked(ThrowingRunnable<?> r) {
     }
   }
 
+  public interface ThrowingFunction<T, R, E extends Exception> {
```

Comment:
Can we please add comments to this method to reduce the cognitive complexity of this method

Reply:
Added Javadoc for the same. Done.